src/Dockerfile

```diff
@@ -1,6 +1,12 @@
 FROM alpine:latest
 
-LABEL org.opencontainers.image.authors="Antonio Tauro <toni.tauro@adfinis.com>"
+LABEL org.opencontainers.image.title="back8sup" \
+      org.opencontainers.image.description="A simple way to backup your kubernetes resources" \
+      org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>" \
+      org.opencontainers.image.vendor="Adfinis" \
+      org.opencontainers.image.url="https://github.com/adfinis-sygroup/back8sup" \
+      org.opencontainers.image.licenses="AGPL-3.0-or-lateræ
+
```

Comment:
```suggestion
      org.opencontainers.image.licenses="AGPL-3.0-or-later
```

Reply:
My suggestion doesn't seem to make it clear that I am also nitpicking on the addition of the empty line 9.